### PR TITLE
update: added landing page

### PR DESCRIPTION
**Pull request by simran61**
Screenshot (2003)
Screenshot (2004)
Screenshot (2005)
Screenshot (2006)

### CODE REVIEW

**Review by ikkurthis1998 — APPROVED** [review]
I can see that you are not using any components from your component library. My suggestion is if your component library doesn't have any component, once you make the component here, add it to component library as well. This way you can get a richer component library by the end of the camp

File: style.css [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+    align-items: center;
+    min-width: 29rem;
+    display: flex;
+    justify-content: space-between;
+    padding: 0.5rem 0;
+}
+
+.popular_icon{
+    height: 70px;
+}
```
Comment (Tarandeep-s1ngh) [inline style.css:241]: Better to use relative units like rems over here

**Review by ragini07 — APPROVED** [review]
Looks good

File: style.css [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+
+.card:hover{
+    transition: 0.5s;
+    box-shadow: 8px 8px 10px #888888;
+}
+
+.quiz_icon{
+    position: absolute;
+    height: 120px;
```
Comment (Tarandeep-s1ngh) [inline style.css:281]: Again, Better to use relative units like rems over here

**Review by abhisekgupta001 — APPROVED** [review]
Awesome UI. The use of 3D illustrations with gradients just elevated your design. Great work.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+    height: 200px;
+}
+
+.lb_stats{
+    display: flex;
+}
+
+.person_img{
+    height: 80px;
+    width: 80px;
```
Comment (Tarandeep-s1ngh) [inline style.css:386]: Better to use relative units like rems over here

**Review by Tarandeep-s1ngh — APPROVED** [review]

**Review by ayush65 — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (simran61) [reply style.css:386]: okk i'll update

File: style.css (hunk 2 above)
Comment (simran61) [reply style.css:281]: sure

**Pull request merged**